constraints.md

```diff
@@ -251,6 +251,8 @@ To achieve better performance of pessimistic transactions, you can set the [`tid
     ERROR 9007 (HY000): Write conflict, txnStartTS=435688780611190794, conflictStartTS=435688783311536129, conflictCommitTS=435688783311536130, key={tableID=74, indexID=1, indexValues={bill, }} primary={tableID=74, indexID=1, indexValues={bill, }}, reason=LazyUniquenessCheck [try again later]
     ```
 
+- When this variable is disabled, if there is a write conflict among multiple pessimistic transactions, the pessimistic lock may be forced to roll back when the other pessimistic transactions commit, thus generating a `Pessimistic lock not found` error. When this error occurs, it means that the application scenario is not suitable for deferring the unique constraint check of the pessimistic transaction, and you should consider adjusting the application logic to avoid the conflict or retrying the transaction after the error occurs.
```

Comment:
```suggestion
- When this variable is disabled, if there is a write conflict among multiple pessimistic transactions, the pessimistic lock might be forced to roll back when other pessimistic transactions are committed, thus resulting in a `Pessimistic lock not found` error. When this error occurs, it means that deferring the unique constraint check of the pessimistic transaction is not suitable for your application scenario. In this case, consider adjusting the application logic to avoid the conflict or retrying the transaction after getting the error.
```